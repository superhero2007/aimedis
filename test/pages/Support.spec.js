import React from 'react';
import { shallow } from 'enzyme';

import Support from 'pages/Support';

function setup() {
  return shallow(<Support />);
}

describe('Support', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();

    const activeButtonElement = wrapper.find('.app__support__section1__content__form__button');
    activeButtonElement.simulate('click');

    expect(wrapper.html()).toMatchSnapshot();
  });
});
