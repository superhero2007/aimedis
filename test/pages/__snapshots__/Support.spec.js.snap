// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Support should render properly 1`] = `"<div class=\\"app__support app__route\\"><div class=\\"container\\"><section class=\\"app__support__section1\\"><div class=\\"app__support__section1__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-6 offset-lg-3\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__support__section1__content__title\\">Support Message</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__support__section1__content__description\\">If you have a question or are experiencing issues with Aimedis, you can send us a Sopprt Message and we will back to you ASAP</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__support__section1__content__form\\"><div class=\\"app__input\\"><label class=\\"app__input__label\\">Name</label><input type=\\"text\\" class=\\"app__input__text\\" style=\\"height:3.5rem\\"/></div><div class=\\"app__input\\"><label class=\\"app__input__label\\">E-mail address</label><input type=\\"text\\" class=\\"app__input__text\\" style=\\"height:3.5rem\\"/></div><div class=\\"app__input\\"><label class=\\"app__input__label\\">Message</label><input type=\\"text\\" class=\\"app__input__text\\" style=\\"height:11.125rem\\"/></div><button type=\\"button\\" class=\\"btn btn__primary app__support__section1__content__form__button\\">SEND</button></div></div></div></div></div></section></div></div>"`;

exports[`Support should render properly 2`] = `"<div class=\\"app__support app__route\\"><div class=\\"container\\"><section class=\\"app__support__section1\\"><div class=\\"app__support__section1__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-6 offset-lg-3\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__support__section1__content__title\\">Support Message</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__support__section1__content__description\\">If you have a question or are experiencing issues with Aimedis, you can send us a Sopprt Message and we will back to you ASAP</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__support__section1__content__form\\"><div class=\\"app__input\\"><label class=\\"app__input__label\\">Name</label><input type=\\"text\\" class=\\"app__input__text\\" style=\\"height:3.5rem\\"/></div><div class=\\"app__input\\"><label class=\\"app__input__label\\">E-mail address</label><input type=\\"text\\" class=\\"app__input__text\\" style=\\"height:3.5rem\\"/></div><div class=\\"app__input\\"><label class=\\"app__input__label\\">Message</label><input type=\\"text\\" class=\\"app__input__text\\" style=\\"height:11.125rem\\"/></div><button type=\\"button\\" class=\\"btn btn__primary app__support__section1__content__form__button\\">SEND</button></div></div></div></div></div></section><section class=\\"app__support__section2 d-none d-lg-block\\"><div class=\\"app__support__notification\\"><p class=\\"app__support__notification__context\\">Support Message was send succesfuly!</p><div class=\\"app__support__notification__icon\\"><i class=\\"i-check-circle-o\\"></i></div></div></section></div></div>"`;
